CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # 17.1.0 -
 ## Fixes
+- ISY-618: Incomplete-Date-Widget akzeptiert auch normale Dateneingabe mit Punkt
```

Comment:
slightly unclear for the user what the error was and what is happening now

Reply:
Change log is updated